cron/GET.py

```diff
@@ -5,7 +5,7 @@
 import hashlib
 import sys
 import subprocess
-import tunnelcrypt
+import cryptex
```

Comment:
I think this should pe
```
from ping_me import cryptex
```

Maybe we should put `cryptex.py` inside somewhere `ping_me/utils` and import it from there. What say?